VERSION

```diff
@@ -1 +1 @@
-1.1.0
\ No newline at end of file
+1.2.0
```

Comment:
I want to consult with @balteravishay regarding this... not sure if that's what we want :) by the time the project will be out we will be at version 1.100.0

Reply:
We can keep the discussion in the teams

Reply:
@shiranr So minor it is, leaving it as is.